src/cmd/builtin/restore.c

```diff
@@ -56,6 +60,13 @@ static void progress (int delta_blob, int delta_keys)
                  keycount,
                  blobcount);
     }
+#if HAVE_LIBSYSTEMD
+    if (sd_notify_flag
+        && (keycount % 100 == 0 || keycount < 10)) {
+        sd_notifyf (0, "EXTEND_TIMEOUT_USEC=%d", 10000000); // 10s
+        sd_notifyf (0, "STATUS=flux-restore(1) has restored %d keys", keycount);
```

Comment:
Does `flux-restore(1)` know the total number of keys being restored? If so it might be useful to supply that here as a kind of progress indicator.

Reply:
That makes sense. Perhaps in the future `flux-dump` could write a final item to the archive in a well known path that gives the total number of keys if it becomes an issue. 

Actually probably `flux dump` is the bigger issue. On a recent system the dump took >6m -- it might be useful for admins to know how far along it is. Not sure if that one's even possible though.

Anyway, this isn't really applicable to this PR, so nevermind.